airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1419,12 +1397,15 @@ components:
 
     ConnectionCollection:
       type: object
-      description: Connections
-      properties:
-        connections:
-          type: array
-          items:
-            $ref: '#/components/schemas/ConnectionCollectionItem'
+      description: Collection of connections.
+      allOf:
+       - type: object
+         properties:
+           connections:
+            type: array
```

Comment:
You have linting errors in ConnectionCollection. Can you install pre-commit so you can find issues locally before committing. https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst#pre-commit-hooks.

This should work:
```
    ConnectionCollection:
      type: object
      description: Collection of connections.
      allOf:
        - type: object
          properties:
            connections:
              type: array
              items:
                $ref: '#/components/schemas/ConnectionCollectionItem'
        - $ref: '#/components/schemas/CollectionInfo'
```
If you accept, I can push a fix for you but first try and see if you are able to resolve it. Thanks

Reply:
done :rocket: 
sorry about that, planned to do it after the weekend :grin: